### PR TITLE
Co-locates tests with source files

### PR DESCRIPTION
**Pull request by jamonholmgren**
:tada:

### CODE REVIEW

File: packages/gluegun/src/cli/parse-command-line.test.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const test = require('ava')
-const parseCommandLine = require('../../src/cli/parse-command-line')
+const parseCommandLine = require('./parse-command-line')
```
Comment (jamonholmgren) [inline packages/gluegun/src/cli/parse-command-line.test.js]: This is so pretty.

File: packages/gluegun/src/utils/filesystem-utils.test.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ test('isNotDirectory', t => {
 
 test('subdirectories', t => {
   const dirs = subdirectories(`${__dirname}/..`)
-  t.is(6, dirs.length)
+  t.is(dirs.length, 6)
```
Comment (jamonholmgren) [inline packages/gluegun/src/utils/filesystem-utils.test.js]: Ava expects the hard-coded value to be on the right.

Comment (skellock) [reply packages/gluegun/src/utils/filesystem-utils.test.js]: yep.  `actual` and `expected`.

**Pull request merged**